app/controllers/api/v1/versions/downloads_controller.rb

```diff
@@ -1,35 +0,0 @@
-class Api::V1::Versions::DownloadsController < Api::BaseController
```

Comment:
Yeah I guess we should try to be a little nice, let's just render a simple message saying these endpoints are not available anymore.